crates/turbopack-browser/src/ecmascript/list/version.rs

```diff
@@ -19,6 +21,27 @@ pub(super) struct EcmascriptDevChunkListVersion {
     pub by_merger: IndexMap<Vc<Box<dyn VersionedContentMerger>>, VersionTraitRef>,
 }
 
+fn eq_index_map<K: Hash + Eq, V>(
+    this: &IndexMap<K, TraitRef<V>>,
+    other: &IndexMap<K, TraitRef<V>>,
+) -> bool {
+    if this.len() != other.len() {
+        return false;
+    }
+
+    this.iter()
+        .all(|(key, value)| other.get(key).map_or(false, |v| value.ptr_eq(v)))
```

Comment:
Order should matter here. You can use `zip()` instead. That's also more performant.